ompi/attribute/attribute_predefined.c

```diff
@@ -104,34 +107,54 @@ static int free_win(int keyval);
 
 static int set_f(int keyval, MPI_Fint value);
 
-
-int ompi_attr_create_predefined(void)
+int ompi_attr_create_predefined_keyvals(void)
```

Comment:
This function is always called while the instance_lock mutex is held. It is really necessary to protect it again with attrs_predefined_lock ?

Reply:
good point.  removing.